cypress/e2e/the-internet/the-internet-tests.cy.js

```diff
@@ -270,4 +271,17 @@ describe("The Internet Test Suite", () => {
 		floatingMenuPage.scrollToBottom();
 		floatingMenuPage.getMenu().should("be.visible");
 	});
+
+	it("Tests Hovers", () => {
+		const hoversPage = new hovers();
+		for (var index = 0; index < 3; index++) {
+			hoversPage.visit();
+			hoversPage.getProfileInfo().eq(index).should("not.be.visible");
+			hoversPage.getViewProfileButton().eq(index).should("not.be.visible");
+			hoversPage.getProfileInfo().eq(index).invoke("show");
+			hoversPage.getProfileInfo().eq(index).should("be.visible");
+			hoversPage.getViewProfileButton().eq(index).should("be.visible");
+			hoversPage.getViewProfileButton().eq(index).click();
```

Comment:
Maybe try adding a check for the not found page for each click before it continues, at its current state the test sometimes does not pass.  Something like **_hoversPage.getNotFoundTitle().should('be.visible')_**

Reply:
Added the necessary class